packages/client/hmi-client/src/components/navbar/tera-navbar.vue

```diff
@@ -217,12 +268,37 @@ const persistEvaluationScenario = () => {
 	window.localStorage.setItem('evaluationScenarioNotes', evaluationScenarioNotes.value);
 };
 
-const loadEvaluationScenario = () => {
+const refreshEvaluationScenario = async () => {
+	evaluationScenarioCurrentStatus.value = (
+		await API.get(`/evaluation/status?name=${evaluationScenarioName.value}`)
+	).data;
+
+	evaluationScenarioRuntimeMillis.value = (
+		await API.get(`/evaluation/runtime?name=${evaluationScenarioName.value}`)
+	).data;
+};
```

Comment:
Those calls should usually be under `src/services` file, but I guess this can be done in an eventual navbar cleanup